README.md

```diff
@@ -135,6 +143,19 @@ def deps do
 end
 ```
 
+## Configuration
+
+```elixir
+# This is the default config, even if not configured.
+config DateTimeParser, parsers: [:epoch, :serial, :tokenizer]
+
+# To disable some parsers, exclude it from the :parsers key.
```

Comment:
I wonder if there is value in having a whitelist and a blacklist. 

Can you imagine a scenario where someone would want to specify parsers to use, and be sad if a new parser was added and they weren't automatically opted-in to using it, or do you imagine if people are using this option it's to only use (a) specific parser(s) and they don't want to use anything else?

Reply:
Good thought! I imagine the later scenario, and when they upgrade the library hopefully they'll consult the changelog to see if we've added new parsers, and include it if they choose to.